### ISSUE TITLE
Issue running Dev Container

### ISSUE THREAD

**Issue by farazfazli** [purton-tech/rust-on-nails#39]
Hi Ian,

I am facing the following intermittent issue while trying to run the Dev Container: `WebSocket Protocol Error: Unable to parse WebSocket key.`

System Details: Ubuntu 22.10 w/ AMD Ryzen Processor

**Command run, after ensuring Git submodules are updated**
`cd rust-on-nails.com && zs`

**Terminal**
![terminal](https://user-images.githubusercontent.com/8684210/226211553-7f439927-8b40-42ca-96e0-e74209a1fb68.png)

**Ports**
![ports](https://user-images.githubusercontent.com/8684210/226211555-d3be5945-eaab-4482-9615-eecb0e16ea89.png)

Sometimes it works and sometimes it doesn't. Please let me know if you need any additional details from my end.

Thanks,
Faraz

**Comment by ianpurton** [comment]
I'm also on Ubuntu on a ryzen processor. I've only been using this setup for a few days so we'll see if I get this issue too.

```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.10
Release:	22.10
Codename:	kinetic
```

This issue could be that when you run zola it creates a websocket connection back to the server to wait for updates. Then it automatically refreshes when it sees that update.

I couldn't find a way to disable this.

**Comment by ianpurton** [comment]
So I fix the version of Zola at 0.15.3, this is because that's the version that cloudflare supports.

You may have better luck with a later version

```Dockerfile
ARG ZOLA_VERSION=0.15.3
RUN sudo curl -OL https://github.com/getzola/zola/releases/download/v$ZOLA_VERSION/zola-v$ZOLA_VERSION-x86_64-unknown-linux-gnu.tar.gz \
    && sudo tar -xvf zola-v$ZOLA_VERSION-x86_64-unknown-linux-gnu.tar.gz \
    && sudo mv zola /usr/bin/zola \
    && sudo chmod +x /usr/bin/zola
```

**Comment by farazfazli** [comment]
Looks like it's working now, thank you!
